From 3cfc0933c38a9f5383042347ef659c3d982090a3 Mon Sep 17 00:00:00 2001
From: Stephan Bergmann <sbergman@redhat.com>
Date: Wed, 8 Oct 2014 13:22:44 +0200
Subject: [PATCH] libgcrypt and gnutls are only used by our internal
 external/neon

...so must not be checked for if --without-system-neon

Change-Id: If598108f98d6f94f32650feba7044224a78a4c12
(cherry picked from commit c592e7dea2e5ac326041b78b9512600c984ccced)
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 363eb2f..ed3da6c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -9463,7 +9463,7 @@ dnl ===================================================================
 dnl Check for building gnutls
 dnl ===================================================================
 AC_MSG_CHECKING([whether to use gnutls])
-if test "$WITH_WEBDAV" = "neon" && test "$enable_openssl" = "no"; then
+if test "$WITH_WEBDAV" = "neon" -a "$with_system_neon" = no -a "$enable_openssl" = "no"; then
     AC_MSG_RESULT([yes])
     AM_PATH_LIBGCRYPT()
     PKG_CHECK_MODULES(GNUTLS, [gnutls],,
-- 
1.9.3

